packages/core/echo/echo-schema/src/automerge/automerge-db.ts

```diff
@@ -218,36 +231,125 @@ export class AutomergeDb {
     }
   }
 
+  private _initDocAccess(handle: DocHandle<SpaceDoc>) {
+    handle.change((newDoc: SpaceDoc) => {
+      newDoc.access = { spaceKey: this.spaceKey.toHex() };
```

Comment:
Even though this only runs when `acesss === null`, doing it this way still feels a bit safer:

```suggestion
      newDoc.access ??= {};
      newDoc.access.spaceKey = this.spaceKey.toHex();
```

what do you think?

Reply:
I don't mind applying the suggestion, just curious why does it feel safer?

Reply:
I imagine future versions might store more info inside `access`. This code would delete every field inside `access`

Reply:
My conceptual model was that everything in the SpaceDoc up until user data is treated as a key-value store. So intermediate objects like `access` are just lazily initialized on write but never overriden

Reply:
> I imagine future versions might store more info inside access. This code would delete every field inside access

that's a sound argument 👍